docker/Dockerfile

```diff
@@ -10,7 +10,7 @@
 # such information to foreign countries or providing access to foreign
 # persons.
 
-FROM python:3.8-slim
+FROM python:3.9-slim
```

Comment:
You have 3.9 here, but in the .toml file you specifiy ^3.8.  
Shouldn't they be consistent?

Reply:
@frankinspace ^
Also, why did you jump the version from 1.5 to 2.0?

Reply:
`^3.8` means it should run on 3.8 or higher. The reason for switching to 3.9 in our docker image was just because there are less open security issues against the 3.9 image.

2.0 bump is due to the breaking change that we no longer support python 3.7